_examples/render/main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"math/rand"
+	"net/http"
+	"reflect"
+	"time"
+
+	"github.com/pressly/chi"
+	"github.com/pressly/chi/_examples/render/data"
+	"github.com/pressly/chi/_examples/render/presenter/v1"
+	"github.com/pressly/chi/_examples/render/presenter/v2"
+	"github.com/pressly/chi/_examples/render/presenter/v3"
+	"github.com/pressly/chi/middleware"
+	"github.com/pressly/chi/render"
+)
+
+func main() {
+	r := chi.NewRouter()
+
+	r.Use(middleware.RequestID)
+	r.Use(middleware.Logger)
+	r.Use(middleware.Recoverer)
+	r.Use(render.ParseContentType)
```

Comment:
do we still need render.ParseContentType() ..? perhaps we can rename this or do something else to get the requested content-type?